Elevel/Application/Features/AuditionFeatures/CreateAuditionCommand.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using AutoMapper;
+using Elevel.Application.Interfaces;
+using Elevel.Domain.Models;
+using MediatR;
+
+namespace Elevel.Application.Features.AuditionFeatures
+{
+    public class CreateAuditionCommand
+    {
+        public class Request : IRequest<Response>
+        {
+            public Guid Id { get; set; }
```

Comment:
Where do you pick the data from?
You create the Audition, but you don't receive anything from the request

Reply:
Why do you need Id field in Create flow?